File: pset_1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // getting user input
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    // repeat space and hash printing for each row
    for (int i = 1; i <= height; i++)
    {
        // print spaces
        for (int spaces = i; spaces < height; spaces++)
        {
            printf(" ");
        }

        // print hashes
        for (int hashes = 0; hashes <= i; hashes++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset_6/sentimental-mario-less/mario.py
import cs50

# wanting to continue running until we get a valid input
while True:
    user_input = int(cs50.get_int(("Height: ")))
    if user_input <= 8 and user_input >= 1:
        break

# using user input to set the range for the loop
for i in range(user_input):
    print(" " * (user_input - i - 1), end="")
    print("#" * (i + 1), end="")
    print()
